### DISCUSSION TITLE
Some weird errors when deleting or saving multiple times a file

### DISCUSSION THREAD

**Discussion by MrRoiz** [folke/noice.nvim#439] [Q&A]
Hey there, I just tried this plugin (looks awesome btw) but I got 2 issues
1. when I delete a file that was opened in a buffer with nvim-tree mappings, when the file is deleted the buffer closes but if I try to toggle once again nvim-tree I got this error 
![image](https://user-images.githubusercontent.com/49358353/233747910-b7aafe81-9525-414e-ab9c-54102a1b96f4.png)

2. When I save multiple times a file the first notification works perfectly but second and next ones I got this error notification
![image](https://user-images.githubusercontent.com/49358353/233748050-ae5017e4-f64a-4306-a677-7b0cfd8b9aa3.png)

I'm using nvim v0.9 and like a comment, those issues don't happen without using noice.nvim

**Comment by folke** [comment]
Your first error seems to be a neo-tree issue. The second error is a Neovim bug that sometimes happens.
